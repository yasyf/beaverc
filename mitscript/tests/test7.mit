z = 1;
f2 = fun() {
  global z;

  z = z + 1;
  print(z);
};
f2();
