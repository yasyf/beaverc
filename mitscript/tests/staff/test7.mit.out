1
2
3
Bigger 2
Bigger 3
Bigger 4
Bigger a1
Bigger b1
Bigger c1
{toStr:FUNCTION toStr_:FUNCTION tail:FUNCTION head:FUNCTION }
[1, 2, 3, ]
